src/gui/org/deidentifier/arx/gui/view/impl/menu/DialogOrderSelection.java

```diff
@@ -199,7 +199,7 @@ private boolean isValidDataType(DataType<?> type, String[] values) {
      * @param charset TODO
      * @return
      */
-    private String[] loadArray(String file, Charset charset) {
+    private ArrayList<String> loadFile(String file, Charset charset) {

```

Comment:
I explicitly used "ArrayList" as I use the functionality "retainAll" in the selection listener for the import button, which is optional for the List interface (see: https://docs.oracle.com/javase/8/docs/api/java/util/List.html#retainAll-java.util.Collection-)